Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
-name = "bma423"
-version = "0.0.1"
-edition = "2021"
 authors = ["Pierre-Yves Aillet <pyaillet@gmail.com>"]
 categories = ["embedded", "hardware-support"]
 description = "Rust driver for Bma423 accelerometer"
-repository = "https://github.com/pyaillet/bma423-rs"
+edition = "2021"
 license = "MIT OR Apache-2.0"
+name = "bma423"
 readme = "README.md"
-
+repository = "https://github.com/pyaillet/bma423-rs"
+version = "0.0.1"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-embedded-hal = "^0.2"
-num_enum = { version = "^0.5", default-features = false }
-accelerometer = { version = "^0.12", optional = true }
-bitmask-enum = { version = "^1.1" }
+accelerometer = {version = "^0.12", optional = true}
+bitmask-enum = {version = "^1.1"}
+derive-new = "0.5"
```

Comment:
Is this a strong requirement on your side?

I am not a fan of introducing new dependencies if it can be avoided.

Reply:
It looks like the only actual new dependency I added is `derive-new`, noting that I also updated the version of `embedded-hal`. The new dependency is just basically a macro that allows automatically deriving a constructor method (i.e. a `new` function). This is definitely not a strong requirement at all and I have no problem with removing it in the interest of minimizing dependencies. It's only used in one place, on the `Config` struct, and can be easily replaced with a trivial manual `new` constructor. This struct also has all public fields, so maybe doesn't really even need a constructor probably.

Reply:
Added a new commit that removes the `derive-new` dependency.